delphi_srr2_radar_driver_wrapper/src/delphi_srr2_radar_driver_wrapper.cpp

```diff
@@ -95,15 +100,17 @@ void DelphiSrr2RadarDriverWrapper::detection_cb(const radar_msgs::RadarDetection
 
 void DelphiSrr2RadarDriverWrapper::publish_object_track()
 {
-    if(track_msg_ == 0)
+    if(track_msg_ == nullptr)
         return;
     radar_msgs::RadarTrackArray tracks;
-    tracks.header.stamp = ros::Time::now();
-    tracks.header.frame_id = sensor_frame_;
+    tracks.header.stamp = track_msg_->header.stamp;
+    tracks.header.frame_id = track_msg_->header.frame_id;
```

Comment:
If we use my suggestion we will need to keep the seq number too. If it isn't set then set it yourself in the wrapper.

Reply:
It is set by default I think. So I just need to overwrite it in the wrapper.

Reply:
Changed to 50 Hz. Another part of this change is in its base class: https://github.com/usdot-fhwa-stol/CARMADriverUtils/pull/4